api/src/pcapi/routes/backoffice/templates/offerer/get.html

```diff
@@ -174,7 +174,18 @@ <h5 class="modal-title">Supprimer la structure {{ offerer.name }}</h5>
               <p class="mb-1">
                 <a href="{{ offerer | pc_pro_offerer_offers_link }}"
                    target="_blank"
-                   class="fw-bold link-primary">Offres associées</a>
+                   class="fw-bold link-primary">
+                  Offres associées
+                  <i class="bi bi-box-arrow-up-right"></i>
+                </a>
+              </p>
+              <p class="mb-1">
+                <a href="{{ url_for("backoffice_web.offer.list_offers") }}?offerer={{ offerer.id }}"
+                   class="fw-bold link-primary">Offres</a>
```

Comment:
Le ticket dit « Offres BO » ; en attendant d'avoir une refonte UX pour remettre de l'ordre dans les liens et actions, je suis partisan de clarifier (pareil en-dessous)
```suggestion
                   class="fw-bold link-primary">Offres BO</a>
```